presto-spi/pom.xml

```diff
@@ -53,6 +53,12 @@
             <artifactId>jol-core</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>joda-time</groupId>
+            <artifactId>joda-time</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
```

Comment:
remove the dependency. Check my other comment

Reply:
Right, I was under the impression that we should not add this dependency here since SPI should use as less dependencies as possible.